app/controllers/application_controller.rb

```diff
@@ -8,6 +8,11 @@ class ApplicationController < ActionController::Base
 
   private
 
+  def kudosbot
+    # Initialize kudosbot.
```

Comment:
I feel like line 13 is pretty readable, so this comment can probably be removed.